eng/pipelines/build.yml

```diff
@@ -26,27 +39,48 @@ jobs:
       - script: |
           npm ci
         displayName: "Install packages"
+        workingDirectory: $(Build.SourcesDirectory)/autorest.csharp
       - script: |
           dotnet build AutoRest.CSharp.V3.sln
         displayName: "Build"
+        workingDirectory: $(Build.SourcesDirectory)/autorest.csharp
         env:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
       - pwsh: ./eng/CodeChecks.ps1
         displayName: "Check if code is up-to-date"
+        workingDirectory: $(Build.SourcesDirectory)/autorest.csharp
       - script: |
           dotnet test AutoRest.CSharp.V3.sln
         displayName: "Test"
         env:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
+        workingDirectory: $(Build.SourcesDirectory)/autorest.csharp
       - ${{if and(ne(variables['Build.Reason'], 'PullRequest'), eq(variables['System.TeamProject'], 'internal'))}}:
+        - checkout: azure-sdk-for-net
         - pwsh: ./eng/PublishRelease.ps1 -WorkingDirectory ./artifacts/bin/AutoRest.CSharp.V3/Debug/netcoreapp3.0/ -PackageJson ./src/AutoRest.CSharp.V3/package.json -Token $(azuresdk-github-pat) -Artifacts $(Build.ArtifactStagingDirectory) -BuildNumber $(Build.BuildNumber) -Sha $(Build.SourceVersion) -CoverageUser $(storage-coverage-user) -CoveragePass $(storage-coverage-pass)
```

Comment:
I'd suggest folding these line to help with readability. You can fold it by adding a backtick at the end or you can use yml folding by using `>` (https://yaml-multiline.info/)